firebase_read_block.py

```diff
@@ -0,0 +1,20 @@
+from .firebase_base import FirebaseBase
+
+
+class FirebaseRead(FirebaseBase):
+
+    def process_signals(self, signals):
+        out_sigs = []
+        for sig in signals:
+            # The collection may be different for different signals,
+            # so we need to compute it inside the loop
```

Comment:
I agree that the comment blocks aren't necessary. These were copied over from the insert block as it had very similar functionality. I'll remove them from this block but would you recommend that I take them out of the insert block as well?